config/manager/kustomization.yaml

```diff
@@ -1,2 +1,3 @@
 resources:
-- manager.yaml
+- manager-deployment.yaml
+- speaker-daemonset.yaml
```

Comment:
I would just call those files `manager.yaml` and `speaker.yaml`.

Reply:
Sgtm, changed names.